app/src/main/java/org/mozilla/focus/browser/BrowserFragment.kt

```diff
@@ -225,11 +225,6 @@ class BrowserFragment : EngineViewLifecycleFragment(), Session.Observer {
 
         if (session.url == APP_URL_HOME) {
             browserOverlay?.visibility = View.VISIBLE
-        } else {
```

Comment:
We also need to do #1005: since the WebView survives, it needs to have `clearData` called on it after we clear data in the settings (we need a test for that...).

This could be a follow-up but needs to happen for v3.0.

Reply:
Reopened #1005.